stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -1505,16 +1497,15 @@ func.func @dot_bad_precision_config(%arg0: tensor<2x2xi32>, %arg1: tensor<2x2xi3
 
 // -----
 
-func.func @dot_illegal_input_type(%arg0: tensor<3xf32>, %arg1: tensor<?x3xf32>) -> tensor<?xf32> {
-  // expected-error@+1 {{Unexpected result type: has 'tensor<?xf32>' but inferred 'tensor<3xf32>' from operands 'tensor<3xf32>' and 'tensor<?x3xf32>}}
+func.func @dot_more_dynamic_output_type(%arg0: tensor<3xf32>, %arg1: tensor<?x3xf32>) -> tensor<?xf32> {
```

Comment:
It seems that you keep the tests as they are there in https://github.com/tensorflow/tensorflow/blob/82c38106d03c15efe3ee53401e3527348a62a8cb/tensorflow/compiler/xla/mlir_hlo/tests/Dialect/mhlo/ops.mlir#L1099.

No action.

Reply:
The above comment was for removing the test related to DotOp precision_config since the recent change in StableHLO #441, once migrated, will fail in MHLO since DotOp has already been removed there and there are currently discrepancies between the two, which #778 will aim to fix.